rooms/script.js

```diff
@@ -0,0 +1,38 @@
+class room {
+  constructor(name, length, width, number) {
```

Comment:
You didn't add a default value for room capacity. I would have written your constructor parameters as:

```(name, length, width, capacity = 15)```

Putting a parameter = 15 makes 15 the default value. You won't have to pass a value in when you call the class, but if you do, it will overwrite the 15.